src/www/templates/index.html

```diff
@@ -46,7 +46,7 @@ <h1 id="overview" class="page-header">What is DataHub?</h1>
         <li>A database-agnostic, language-agnostic, <a href="https://github.com/mitreid-connect/">MITREid (OpenID + OAuth2)</a> integrated platform for your mobile and web apps</li>
         <li>A web client for easy manipulation of your data</li>
         <li>An application ecosystem for data processing, including ingestion, curation, integration, discovery, query, analytics, visualization, and machine learning</li>
-        <li>An open source project from <a href="https://csail.mit.edu">MIT CSAIL's</a> <a href="livinglab.mit.edu">Living Lab</a></li>
+        <li>An open source project from <a href="csail.mit.edu">MIT CSAIL's</a> <a href="livinglab.mit.edu">Living Lab</a></li>
```

Comment:
These hrefs need to start with a scheme, or they'll be interpreted as relative paths.